helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
+  namespace: {{ $.Values.namespace }}
+  labels:
+    name: {{ .Release.Name }}-app
+    app.kubernetes.io/name: {{ .Release.Name }}-app
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+
+  fabric-network.json: |-
+      {
+      "name": "fabric-network",
+      "version": "1.0.0",
+      "client": {
+        "tlsEnable": true,
+        "adminCredential": {
+          "id": "{{ .Values.explorer.username  | default "exploreradmin"}}",
+          "password": "{{ .Values.explorer.password | default "exploreradminpw"}}"
+        },
+        "enableAuthentication": true,
+        "organization": "{{ .Values.network.mspid}}",
+        "connection": {
+          "timeout": {
+            "peer": {
+              "endorser": "300"
+            },
+            "orderer": "300"
+          }
+        }
+      },
+      "channels": {
+        "{{ .Values.network.channel}}": {
+          "peers": {
+            "{{ .Values.network.peername}}": {}
+          },
+          "connection": {
+            "timeout": {
+              "peer": {
+                "endorser": "6000",
+                "eventHub": "6000",
+                "eventReg": "6000"
+              }
+            }
+          }
+        }
+      },
+      "organizations": {
+        "{{ .Values.network.mspid}}": {
+          "mspid": "{{ .Values.network.mspid}}",
+          "adminPrivateKey": {
+            "pem": "{{ .Values.network.adminPrivateKey}}"
```

Comment:
Please load private key from either a secret store or create in-memory volume to load it from an external secret storage.